version.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>2.2.0</VersionPrefix>
-    <VersionSuffix>rtm</VersionSuffix>
+    <VersionPrefix>2.2.1</VersionPrefix>
+    <VersionSuffix>servicing</VersionSuffix>
     <PackageVersion Condition="'$(IsFinalBuild)' == 'true' AND '$(VersionSuffix)' == 'rtm' ">$(VersionPrefix)</PackageVersion>
```

Comment:
You'll also need to update this line, otherwise the final package builds wont be "2.2.1".  I recommend replacing your version.props file with this: https://github.com/aspnet/EntityFrameworkCore/blob/release/2.2/version.props